src/net.cpp

```diff
@@ -1205,11 +1302,14 @@ void CConnman::AcceptConnection(const SOCKET& listen_socket, bool whitelisted) {
         return;
     }
 
-    if (!IsSelectableSocket(hSocket))
-    {
-        LogPrintf("connection from %s dropped: non-selectable socket\n", addr.ToString());
-        CloseSocket(hSocket);
-        return;
+    const char* method = event_base_get_method(m_event_base);
+    if (!method || (strncmp(method, "select", 6) == 0)) {
```

Comment:
It's only needed if select ends up being used, which should be very unlikely on most platforms. I'll add a comment here, and print something to the log so that it's clear which back-end is being used.

Reply:
We could just bail out if select is being used. But not sure what platforms would be affected.